src/shared/components/query/QueryAndDownloadTabs.tsx

```diff
@@ -85,7 +91,7 @@ export default class QueryAndDownloadTabs extends React.Component<IQueryAndDownl
 					<MSKTab id={"advanced"} linkText={"Query"} onTabDidMount={()=>this.setDefaultTab(undefined)}>
                         <QueryContainer onSubmit={this.props.onSubmit} store={this.store}/>
 					</MSKTab>
-					<MSKTab id={QUICK_SEARCH_TAB_ID} linkText={<span>Quick Search <strong className={"beta-text"}>Beta!</strong></span>} hide={!this.props.showQuickSearchTab} onTabDidMount={()=>this.setDefaultTab(QUICK_SEARCH_TAB_ID)}>
+					<MSKTab id={QUICK_SEARCH_TAB_ID} linkText={<span>Quick Search <strong className={"beta-text"}>Beta!</strong></span>} hide={!this.props.showQuickSearchTab} onTabDidMount={()=>{this.setDefaultTab(QUICK_SEARCH_TAB_ID); this.trackQuickSearch();}}>
```

Comment:
@dippindots 	maybe split this element over multiple lines, kinda hard to read

Reply:
this should prolly only be called when google analytics is enabled. There should be some property

Reply:
@inodb this is actually handled upstream.  the trackEvent function will basically just noop if there is no instance of ga.   i think this is a lot cleaner than having to check everywhere we want to track